src/renderer/index.css

```
@@ -366,6 +366,40 @@
+ *
+ * These set size / weight / tracking / transform / line-height
+ * ONLY. Color and opacity stay at the call site because sidebar
+ * chrome (text-sidebar-foreground) and content surfaces
+ * (text-muted-foreground) legitimately differ. Behavioral classes
+ * (padding, hover, focus, truncate, cmdk selectors) also stay at
+ * the call site.
+ * ------------------------------------------------------------ */
+@layer components {
+  .label-section {
+    @apply text-xs font-normal uppercase tracking-wider;
+    line-height: 1.4;
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)